app/serializers/serializers/targets_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class Serializers::TargetsSerializer < Serializers::Base
```

Comment:
This looks more like an helper than a serializer as there are no input arguments and it's not fetching any of the data from the database.
Morover, it's actually using helpers method already; so I would move the entire thing into the helper of the controller that's using it, so this can be used in the view straight away.  

What do you think? @WKocur

Reply:
Makes sense - I was unsure about this already, so will move to a helper.